PF/Assignment1/LibraryHelperClass.java

```diff
@@ -0,0 +1,152 @@
+package ass1;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class LibraryHelperClass {
+	private Connection connection;
+	private ResultSet resultSet;
+	private Statement statement;
+	private Scanner sc = new Scanner(System.in);
+
+	public LibraryHelperClass() {
+		connection = null;
+		resultSet = null;
+		statement = null;
+		ConnectionUtil conUtil = new ConnectionUtil();
+		connection = conUtil.getConnection();
+	}
+
+	// get connection
+	public Connection getConnection() {
+		return connection;
+	}
+
+	// set connection
+	public void setConnection(Connection connection) {
+		this.connection = connection;
+	}
+
+	public List<Title> getBooksDataRelatedToAuthor() {
+		System.out.println("Enter Name of Author");
```

Comment:
ACTION: Can't we pass just input from user as parameter instead take input in helper class?